src/quo2/components/settings/section_label/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo2.components.settings.section-label.view
+  (:require
+    [quo2.theme :as quo.theme]
+    [quo2.components.settings.section-label.style :as style]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]))
+
+(defn- get-text-color
+  [theme blur?]
+  (if blur?
+    colors/white-opa-40
+    (colors/theme-colors colors/neutral-50
+                         colors/neutral-40
+                         theme)))
+
+(defn view
+  "Props:
+   - section - the label of the section
+   - description (optional) - description of the section
+   - blur? (optional) - use blurred styling"
+  [{:keys [section description blur?]}]
+  (let [theme        (quo2.theme/get-theme)
```

Comment:
this is not the desired way to use the theme as this interacts directly with the global ratom, rather we use a react context provider and pass it with a HOC wrapper (`with-theme`). There are many examples in the quo2 components using this approach 👍

Reply:
https://github.com/status-im/status-mobile/blob/develop/src/quo2/README.md#themes

Reply:
Thanks! Didn't see this README before 😊